internal/validators/plain.go

```diff
@@ -0,0 +1,41 @@
+package validators
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+)
+
+func NewPlainValidator() (Validator, error) {
+
+	return &plainValidator{}, nil
+}
+
+type plainValidator struct {
+}
+
+func (r *plainValidator) Validate(bundle fs.FS) error {
+	var validationErrors []error
+	objects, err := GetObjects(bundle)
+	if err != nil {
+		validationErrors = append(validationErrors, fmt.Errorf("get objects from bundle manifests: %v", err))
+	}
+	if len(objects) == 0 {
+		validationErrors = append(validationErrors, errors.New("invalid bundle: found zero objects: "+
+			"plain+v0 bundles are required to contain at least one object"))
+	}
+
+	// TODO(asmacdo) dev only, remove before merge
+	validationErrors = append(validationErrors, errors.New(""))
+	validationErrors = append(validationErrors, errors.New("hello"))
+	validationErrors = append(validationErrors, errors.New("world"))
+	validationErrors = append(validationErrors, errors.New("annnndd"))
+	validationErrors = append(validationErrors, errors.New("four Errors!"))
+	validationErrors = append(validationErrors, errors.New("moved this shit around!"))
+
+	err = errors.New("")
+	for _, each := range validationErrors {
+		err = fmt.Errorf("%v\n%v", err, each)
+	}
```

Comment:
This seems like it will always result in a non-nil error, which seems undesirable. Perhaps consider using https://pkg.go.dev/k8s.io/apimachinery/pkg/util/errors#Aggregate

Reply:
ah thanks, I was looking for errors.Wrap and didn't see it